### ISSUE TITLE
DBeaver Does not display data in DatetimeOffset Columns

### ISSUE THREAD

**Issue by rafaga** [dbeaver/dbeaver#281]
In SqlServer when a table has one or more columns  with datatype DateTimeOffset 
DBeaver shows the data as NULL, but the information is still on table.

DBeaver
id_prioridad |nombre |id_usuario |fecha |
-------------|-------|-----------|------|
1            |Alta   |00B0T      |      |
2            |Media  |00B0T      |      |
3            |Baja   |00B0T      |      |

Microsoft Sql Server Management Studio
1   Alta    00B0T   2015-09-18 10:34:14.5630000 -06:00
2   Media   00B0T   2015-09-18 10:34:23.3930000 -06:00
3   Baja    00B0T   2015-09-18 10:34:30.9100000 -06:00

**Comment by serge-rider** [comment]
Can you send me a sample DDL?

**Comment by rafaga** [comment]
CREATE TABLE Example(
	id tinyint IDENTITY(1,1) NOT NULL,
	sname varchar(50) NOT NULL,
	id_usr varchar(50) NOT NULL,
	fecha datetimeoffset NOT NULL DEFAULT (getdate()),
 CONSTRAINT PK_act_Prioridades PRIMARY KEY CLUSTERED 
(
	id_prioridad ASC
) ON PRIMARY;

**Comment by serge-rider** [comment]
Hmm. I can see datetimeoffset values:
![datetimeoffset](https://cloud.githubusercontent.com/assets/6398845/13756088/45e2b83c-ea2f-11e5-843b-67c415f0f056.png)
It is in SQL Server Express, jtds driver.

What driver do you use? Maybe some specific options?

**Comment by rafaga** [comment]
it is SQL Server 2008 R2 but i used the microsoft Driver, when i switched to jtds, it shows i assume that it is a driver bug -_-

**Comment by rafaga** [comment]
I reproduced but it is a Bug only present in Microsoft driver 

i did this 
 everything executes well until the last query

```
CREATE TABLE Example(
[id] tinyint IDENTITY(1,1) NOT NULL,
sname varchar(50) NOT NULL,
id_usr varchar(50) NOT NULL,
fecha datetime NOT NULL DEFAULT (getdate()),
CONSTRAINT PK_example PRIMARY KEY CLUSTERED
(
[id] ASC
) );

INSERT INTO Example (sname,id_usr) VALUES ('hello','0000a');
INSERT INTO Example (sname,id_usr) VALUES ('bye','0000b');
INSERT INTO Example (sname,id_usr) VALUES ('ktnx','0000c');


SELECT [id], fecha INTO mig_example FROM example;
ALTER TABLE example DROP CONSTRAINT DF__Example__fecha__74444068;
ALTER TABLE example DROP COLUMN fecha;
ALTER TABLE example ADD fecha datetimeoffset DEFAULT(CURRENT_TIMESTAMP) NOT NULL;
```
 at this point if you executes a select on the table show datatimeoffset as null, but only in microsoft driver

```
UPDATE e SET e.fecha = CONVERT(varchar, mig.fecha, 126) + '-06:00', e.id_usr='00B0T'
	FROM Example AS e INNER JOIN mig_Example AS mig ON ( mig.[id] = e.[id] );
DROP TABLE mig_example;
```

**Comment by serge-rider** [comment]
I tried with driver "Microsoft JDBC Driver 4.0 for SQL Server [4.0.2206.100]".
Still can't reproduce. Maybe it not happens with SQL Server Express.

Please check Error Log (can be opened in main menu Window) for any related errors.
If there is nothing interesting - check debug log (home directory .dbeaver/.metadata/dbeaver-debug.log).

**Comment by rafaga** [comment]
Hi checked the files and found this

> 
> !ENTRY org.jkiss.dbeaver.model 2 0 2016-03-10 18:47:09.674
> !MESSAGE Unrecognized type 'microsoft.sql.DateTimeOffset' - can't convert to date/time value

and this is the error log:

**Comment by serge-rider** [comment]
What exact Microsoft JDBC driver version do you use?

**Comment by rafaga** [comment]
Microsoft JDBC Driver 4.0 for SQL Server [4.0.2206.100]

**Comment by krkhnh** [comment]
I'm having this issue with mssql-jdbc-7.0.0.jre8.jar with the same error logs.

**Comment by serge-rider** [comment]
Fixed in #4724
